### PR TITLE
Core14

### PR DESCRIPTION
**Pull request by thebevrishot**
## PR intention
Update parser to support core14 and add more tests.

## Code changes brief
- Create `ZcoinMsgTx` which contain wire.MsgTx to store bitcoin data and extra.
- Update block parser to avoid parsing quorum commit tx with witness flag which is cause of error.
- Update binary version
- Add tests to parse zcoin tx

### CODE REVIEW

File: configs/coins/zcoin.json [hunk 1]
```diff
@@ -22,24 +22,70 @@
...
+      "home/ubuntu/build/zcoin/depends/x86_64-linux-gnu/include/relic_label.h",
+      "home/ubuntu/build/zcoin/depends/x86_64-linux-gnu/include/relic_md.h",
+      "home/ubuntu/build/zcoin/depends/x86_64-linux-gnu/include/relic_pc.h",
+      "home/ubuntu/build/zcoin/depends/x86_64-linux-gnu/include/relic_pool.h",
+      "home/ubuntu/build/zcoin/depends/x86_64-linux-gnu/include/relic_pp.h",
+      "home/ubuntu/build/zcoin/depends/x86_64-linux-gnu/include/relic_rand.h",
+      "home/ubuntu/build/zcoin/depends/x86_64-linux-gnu/include/relic_test.h",
+      "home/ubuntu/build/zcoin/depends/x86_64-linux-gnu/include/relic_trace.h",
+      "home/ubuntu/build/zcoin/depends/x86_64-linux-gnu/include/relic_types.h",
+      "home/ubuntu/build/zcoin/depends/x86_64-linux-gnu/include/relic_util.h",
```
Comment (ultimaweapon) [inline configs/coins/zcoin.json:83]: These files look strange.

File: configs/coins/zcoin.json [hunk 2]
```diff
@@ -22,24 +22,70 @@
...
       "include/bitcoinconsensus.h",
       "lib/libbitcoinconsensus.so",
       "lib/libbitcoinconsensus.so.0",
       "lib/libbitcoinconsensus.so.0.0.0",
+      "README.md",
```
Comment (ultimaweapon) [inline configs/coins/zcoin.json:88]: This too. Why it got packaged?

**Review by ultimaweapon — CHANGES_REQUESTED** [review]

File: configs/coins/zcoin.json (hunk 1 above)
Comment (thebevrishot) [reply configs/coins/zcoin.json:83]: they come together with the release.

File: configs/coins/zcoin.json (hunk 2 above)
Comment (thebevrishot) [reply configs/coins/zcoin.json:88]: same here.

**Review by ultimaweapon — APPROVED** [review]

**Pull request merged**